client/tests/setup/mocks.ts

```diff
@@ -0,0 +1,21 @@
+// To return a different value for each test, you can import this file and use mockReturnValue
+// to set the return value for the function you want to mock using mockReturnValue
```

Comment:
I find this comment a bit unclear--the example shows how to use mockReturnValue, but it doesn't seem to explain how to  change the value for each tests. I see a reset in the `beforeEach`; maybe mention that?

Reply:
Thanks, I think that comment was slightly outdated since I initially had some default values being returned from the functions in this file. I decided that it would cause more trouble than help so I just made them plain `vi.fn()`.

Anyways I've updated this comment and example a bit, hopefully it's more clear if not let me know what would be helpful.